src/Conf.ml

```diff
@@ -253,30 +265,55 @@ end = struct
     in
     let parse = Int.of_string in
     let r = mk ~default:None term in
+    let to_string = Int.to_string in
     let cmdline_get () = !r in
-    let opt = {names; parse; update; cmdline_get; allow_inline; default} in
+    let opt =
+      {names; parse; update; cmdline_get; allow_inline; default; to_string}
+    in
     store := Pack opt :: !store ;
     opt
 
-  let update ~config ~name ~value ~inline =
+  let log_update ~from ~name ~value =
+    match from with
+    | `Attribute -> ()
+    | (`File _ | `Config | `Commandline) as from ->
```

Comment:
nit: `as from` redundant

Reply:
it is not if you want the next match to be exhaustive.

Reply:
Indeed, I missed that.